docs/source/serving/sky-serve.rst

```diff
@@ -308,11 +308,12 @@ Let's bring up a real LLM chat service with FastChat + Vicuna. We'll use the `Vi
       conda activate chatbot
 
       echo 'Starting controller...'
-      python -u -m fastchat.serve.controller > ~/controller.log 2>&1 &
```

Comment:
nit: should we also mention serving on kubernetes a bit in the serving doc and refer to the cloud-permissions for the setup?
We may also want to explicitly list out why people should use skyserve on kubernetes than just using deployments, such as:
1. easier customization for load balancer and autoscaling policy
2. simple interface and easy to setup
3. spillover to clouds.

Reply:
Good idea - filed https://github.com/skypilot-org/skypilot/issues/3518, will submit a separate PR for docs.